package/gluon-config-mode-theme/sass/gluon.scss

```diff
@@ -184,133 +179,144 @@ ul.tabmenu li {
 	list-style: none;
 	margin: 0;
 	padding: 0;
-}
 
-#topmenu li {
-	display: inline-block;
+	li {
+		display: inline-block;
+	}
 }
 
 #maincontent {
-	padding: 0 1em 2em;
-	max-width: 60em;
-	min-width: 40em;
-	margin: 1em auto;
-}
+	max-width: 60rem;
+	margin: 1rem auto;
+	background-color: rgba(255, 255, 255, 0.333);
```

Comment:
The background color change doesn't seem to be covered by the changes listed in the commit/PR description.

Reply:
in another fork i used a bg-image. thats where this 0.333-opacity stuff is from. i'll take that one back.

Reply:
fixed with 96f58d16511d4721ebcdee3be15872d808343810